### PR TITLE
chore: Bump all dependencies

### PR DESCRIPTION
**Pull request by kibertoad**
Since we are close to release, mass-bumping everything.

#### Checklist

- [] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by climba03003 — CHANGES_REQUESTED** [review]

**Review by climba03003 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -167,10 +167,10 @@
     "snazzy": "^9.0.0",
     "split2": "^4.1.0",
     "standard": "^17.0.0-2",
-    "tap": "^16.0.0",
+    "tap": "^16.2.0",
     "tsd": "^0.20.0",
-    "typescript": "^4.5.4",
-    "undici": "^4.11.3",
+    "typescript": "^4.7.2",
+    "undici": "^4.16.0",
```
Comment (Fdawgs) [inline package.json:173]: Isn't this at v5.2.0 atm? Or is this PR just doing minor/hotfix bumps?

**Review by Fdawgs — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (climba03003) [reply package.json:173]: Actually, it is `5.4.0`, and I don't see a big problem on bumping the major of `undici`.
https://github.com/nodejs/undici/releases/tag/v5.0.0

Comment (kibertoad) [reply package.json]: bumped undici

File: package.json [hunk 2]
```diff
@@ -179,16 +179,16 @@
     "@fastify/error": "^3.0.0",
     "@fastify/fast-json-stringify-compiler": "^2.0.0",
     "abstract-logging": "^2.0.1",
-    "avvio": "^8.1.0",
+    "avvio": "^8.1.3",
     "find-my-way": "v6.2.0",
```
[the remark this answers is not in the document]
Comment (kibertoad) [reply package.json]: what was the story behind making it v6.2.0 in the past?

[the remark this answers is not in the document]
Comment (climba03003) [reply package.json]: I think it just unintended change in the PR. it change back from github branch to version.

**Pull request merged**